capture/packet.c

```diff
@@ -329,12 +341,13 @@ LOCAL void moloch_packet_process(MolochPacket_t *packet, int thread)
         }
 
         if (pcapFileHeader.dlt == DLT_EN10MB) {
+            LOG("Adding normal mac to session with 0 offzet direction %d", packet->direction );
             if (packet->direction == 1) {
-                moloch_field_macoui_add(session, mac1Field, oui1Field, pcapData+0);
-                moloch_field_macoui_add(session, mac2Field, oui2Field, pcapData+6);
+                moloch_field_macoui_add_debug(session, mac1Field, oui1Field, 0,0,pcapData+0);
```

Comment:
when you change these and the following lines back, make sure you are consistent on formatting. i prefer ", " however line consistency is the most important.